### PR TITLE
Test for phone number and email

### PR DESCRIPTION
**Pull request by swojeet**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -166,14 +191,20 @@ DEPENDENCIES
   bootstrap-sass (~> 3.3.6)
   byebug
   coffee-rails (~> 4.2)
+<<<<<<< HEAD
+  factory_girl_rails
+=======
   font-awesome-sass
+>>>>>>> 792d29de26e3d98b68bc978f7ada0a0bd18ad3c9
```
Comment (takehiromouri) [inline Gemfile.lock]: looks like there's a merge conflict here - let's fix this :) (just remove the <<<<< and ===== >>>> )

**Comment by takehiromouri** [comment]
Everything looks good to go besides the merge conflict! Awesome job writing the tests!

**Comment by rabingaire** [comment]
I now little about tests right now but i feel the code is perfect :)

**Comment by swojeet** [comment]
merge conflict removed! :)

**Comment by takehiromouri** [comment]
Going to merge this in, I think @sthpravin is quite busy with exams this week :)

**Pull request merged**